### PR TITLE
V14 QA Added acceptance tests for Dictionary

### PR DESCRIPTION
**Pull request by nhudinh0309**
Updated api tests for Dictionary
Added .udt files to test the import dictionary function
Added ui tests for Dictionary
Bumped version of helpers

### CODE REVIEW

**Review by andr317c — CHANGES_REQUESTED** [review]
Had a few comments

**Comment by nhudinh0309** [comment]
Fixed comments

File: tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/Dictionary/Dictionary.spec.ts [hunk 1]
```diff
@@ -2,78 +2,117 @@ import {test} from '@umbraco/playwright-testhelpers';
 import {expect} from "@playwright/test";
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/Dictionary/Dictionary.spec.ts:3]:
```suggestion
import * as crypto from 'crypto';
```
When I ran the API tests for dictionary I got the error ReferenceError: crypto is not defined. So we need to define the import for the test to work

**Review by andr317c — CHANGES_REQUESTED** [review]
One comment

**Pull request merged**